### PR TITLE
SOC-811 Fixing caching issues for i18n on /join page

### PR DESCRIPTION
**Pull request by bkoval**
@rogatty @mklucsarits

### CODE REVIEW

**Comment by rogatty** [comment]
LGTM

File: server/lib/AuthUtils.ts [hunk 1]
```diff
@@ -39,3 +39,7 @@ export function getCacheBusterUrl(redirect: string): string {
 
 	return url.format(cacheBustedUrlObj);
 }
+
+export function disableCache (response: Hapi.Response): void {
```
Comment (hakubo) [inline server/lib/AuthUtils.ts]: There is Caching.ts how about moving it there?

**Comment by mklucsarits** [comment]
I agree with hakubo's comment. Otherwise it looks good to me.

File: server/lib/AuthUtils.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /// <reference path='../../typings/node/node.d.ts' />
-
+/// <reference path='../../typings/hapi/hapi.d.ts' />
```
Comment (mklucsarits) [inline server/lib/AuthUtils.ts]: I know it's not harming anything, but maybe this line can be removed or this file just reverted.

**Comment by mklucsarits** [comment]
LGTM

**Pull request closed without merging**